publications-2020.bib

```diff
@@ -1023,4 +1023,13 @@ @article{maier20f
   pages = {109871},
 }
 
+@mastersthesis{Khi20,
```

Comment:
Not mandatory, but an URL would be nice if one exists. Can this thesis be found anywhere online?

Reply:
Sadly, no.